docs/preview/gha-runner-scale-set-controller/README.md

```diff
@@ -158,6 +158,18 @@ Verify that the secret you provided is correct and that the `githubConfigUrl` yo
 
 ## Changelog
 
+### v0.3.0
+
+#### Major changes
+
+1. Runner pods are more similar to hosted runners (#2348)
```

Comment:
Sadly, GitHub doesn't render these URLs in markdown. So you have manually add them